ffw/BasicCommunicationRPC.js

```diff
@@ -748,7 +748,8 @@ FFW.BasicCommunication = FFW.RPCObserver
           'minute' : current_time.getUTCMinutes(),
           'hour' : current_time.getUTCHours(),
           'day' : current_time.getUTCDate(),
-          'month' : current_time.getUTCMonth(),
+          // Month should be in 1..12 according to HMI_API
```

Comment:
@vlantonov updated in [692f617](https://github.com/CustomSDL/sdl_hmi/pull/5/commits/692f6171d3ef2c9451c3f73f9d2e76f01ebf1572)
This pull request blocks my another request because GetSystemTime related to both OnSystemTimeReady request and starting secure rpc service. It affects only this functionality.